main.js

```diff
@@ -13,6 +13,357 @@ toggle.addEventListener('click', () => {
+
+const cardBtn = document.querySelectorAll('.cardBtn');
+const body = document.querySelector('#portfolio');
+const hero = document.querySelector('.hero');
+const popup = document.querySelector('.popup');
+const collapse = document.querySelector('.fa-x');
+
+cardBtn.forEach((item) => {
+  item.addEventListener('click', () => {
+    popup.classList.add('open');
+    body.classList.add('blur');
+    hero.classList.add('blur');
+  });
+});
+
+collapse.addEventListener('click', () => {
+  popup.classList.remove('open');
+  body.classList.remove('blur');
+  hero.classList.remove('blur');
+});
+
+const demoLink = document.querySelectorAll('.seeLive');
+const source = document.querySelectorAll('.seeLink');
+
+demoLink.forEach((item) => {
+  item.addEventListener('click', () => {
+    window.location.href = gitpages;
+  });
+});
+
+source.forEach((item) => {
+  item.addEventListener('click', () => {
+    window.location.href = github;
+  });
+});
+
+const nameErr = document.querySelector('.fullNameError');
+const emailErr = document.querySelector('.emailError');
+const fullNValid = document.querySelector('.fullNValid');
+const emailValid = document.querySelector('.emailValid');
+const msgErr = document.querySelector('.msgError');
+const msgValid = document.querySelector('.msgValid');
+const submitErr = document.querySelector('.submitErr');
+
+function validateFullName() {
+  const fullName = document.getElementById('fullname').value;
+
+  if (fullName.length === 0) {
+    nameErr.innerHTML = 'Full name is required';
+    fullNValid.innerHTML = null;
+    return false;
+  }
+  if (!fullName.match(/^[A-Za-z]*\s{1}[A-Za-z]*$/)) {
+    nameErr.innerHTML = 'Enter second name';
+    fullNValid.innerHTML = null;
+    return false;
+  }
+  nameErr.innerHTML = null;
+  fullNValid.innerHTML = '<i class="fa-solid fa-circle-check"></i>';
+  return true;
+}
+
+function validateEmail() {
+  const email = document.getElementById('email').value;
+
+  if (email.length === 0) {
+    emailErr.innerHTML = 'Email is required';
+    emailValid.innerHTML = null;
+    return false;
+  }
+  if (
+    !email.match(
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+    )
+  ) {
+    emailErr.innerHTML = 'Email Invalid';
+    emailValid.innerHTML = null;
+    return false;
+  }
+  emailErr.innerHTML = null;
+  emailValid.innerHTML = '<i class="fa-sharp fa-solid fa-circle-check"></i>';
+  return true;
+}
+
+function validateMsg() {
+  const msg = document.getElementById('msg').value;
+  const required = 30;
+
+  const charLeft = required - msg.length;
+  if (charLeft > 0) {
+    msgErr.innerHTML = `${charLeft}more characters required`;
+    msgValid.innerHTML = null;
+    return false;
+  }
+  msgErr.innerHTML = null;
+  msgValid.innerHTML = '<i class="fa-sharp fa-solid fa-circle-check"></i>';
+  return true;
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ Required ] It would be better to fix this linter check instead of disabling it, the reason you are facing this `Eslint` error is because, the `formValidate` function is declared in `main.js` but it's not used in `main.js`. To fix this error I suggest that you refactor your code as shown below
```JavaScript
function formValidate() {
  if (
    !validateFullName() || !validateEmail() || !validateMsg()
  ) {
    submitErr.style.display = 'block';
    submitErr.style.color = 'red';
    submitErr.innerHTML = 'Please fix error to submit';
    setTimeout(() => {
      submitErr.style.display = 'none';
    }, 3000);
    return false;
  }
  submitErr.innerHTML = '<i class="fa-sharp fa-solid fa-circle-check"></i>';
  return true;
}

// These two lines help you fix the linter error
const btn = document.querySelector('button[type=submit]');
btn.addEventListener('click', () => { formValidate(); });
```